crates/db_schema/src/traits.rs

```diff
@@ -215,17 +215,14 @@ pub trait ApubActor: Sized {
     object_id: &DbUrl,
   ) -> impl Future<Output = LemmyResult<Option<Self>>> + Send;
   /// - actor_name is the name of the community or user to read.
+  /// - domain if None only local actors are searched, if Some only actors from that domain
   /// - include_deleted, if true, will return communities or users that were deleted/removed
   fn read_from_name(
     pool: &mut DbPool<'_>,
     actor_name: &str,
+    domain: Option<&str>,
```

Comment:
Combining the `read_from_name` and `read_from_name_and_domain` functions is a good idea, nice.